consent-ws/src/main/java/org/broadinstitute/consent/http/service/DatabaseConsentAPI.java

```diff
@@ -485,4 +479,34 @@ public Election retrieveElection(Integer electionId, String consentId) {
         }
         return election;
     }
+
+    @Override
+    public List<ConsentGroupNameDTO> updateConsentGroupNames(List<ConsentGroupNameDTO> consentGroupNames) {
```

Comment:
I still think the names of this method is misleading as well as updateConsentGroupName. If I just look at the name of this method, it doesn't indicate anything about verification and I might think the updated consents are returned instead returned values indicating an error.

This is one case where in the calling code, it might be more straightforward to call verify separately and return an error if there are values. and then call update.

Reply:
You're right! I didn't realise that with this change is less code and more intuitive.